src/js/videojs-dash.js

```diff
@@ -171,6 +172,16 @@ class Html5DashJS {
 
     this.mediaPlayer_.on(dashjs.MediaPlayer.events.ERROR, this.retriggerError_);
 
+    this.getDuration_ = (event) => {
+      let periods = event.data.Period_asArray;
+
+      if (event.data.mediaPresentationDuration || periods[periods.length - 1].duration) {
```

Comment:
so, if a duration exists, it means this isn't live?

Reply:
Correct. I'll have to double check the spec, but I believe a duration on the currently-active period or overall presentation are mutually exclusive with being live content.

Reply:
Ugh, the spec and IOP are too long and crazy for me to validate this easily, but this is what was suggested by one of the regulars in the dash-if slack.